src/apps/miscapps/generateBaseCellNeighbors.c

```diff
@@ -237,18 +226,19 @@ void generate() {
 
     printf("const int baseCellNeighbors[NUM_BASE_CELLS][7] = {\n");
     for (int i = 0; i < NUM_BASE_CELLS; i++) {
-        char* neighborStrings[7] = {0, 0, 0, 0, 0, 0, 0};
+        printf("{");
```

Comment:
Please undo the changes you made here. This approach will print garbage around the `INVALID_BASE_CELL` printout, instead of only printing that message when reaching that condition, so it will not generate the base cell neighbors code correctly, anymore.

Reply:
Now I'm very confused. The difference between the old code and this is the removal of the `generateBaseCellOutput`, which simply formatted strings that are printed here. Instead, I used identical code here to write the digits/macro directly to stdout.

Reply:
@dfellis Could you paste the output you are seeing or expecting? I ran this branch and got the same output as before.

Reply:
My mistake on the reading of his changes. I did not run his branch.